pyopenms_viz/_config.py

```
@@ -1,15 +1,346 @@
+
+        # Create and return the LegendConfig instance
+        return cls(**config)
+
+    def update_none_fields(self, other: "BaseConfig") -> None:
+        """
+        Update only the fields that are None with values from another BaseConfig object.
+
+        Args:
+        other (BaseConfig): Another BaseConfig object containing values to update.
+        """
+        for field in fields(other):
```

Ruff (0.8.2)

Import `field` from line 2 shadowed by loop variable

(F402)